src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,17 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length > 0 ? (
+      comments.map(comment => (
+        <CommentInfo
+          comment={comment}
+          key={comment.id}
+        />
+      ))
+    ) : (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
```

Comment:
same as with user, better to use CommentList only for CommentList
move logic for `no comments` outside